nixos/modules/services/networking/firewall.nix

```diff
@@ -263,29 +263,29 @@ let
 
   commonOptions = {
     allowedTCPPorts = mkOption {
-      type = types.listOf types.int;
+      type = types.uniq (types.listOf types.int);
```

Comment:
Perhaps these should be `uniq (listOf port)` (and a `with types;` somewhere earlier)

Reply:
(Yes, port is a type)